Model/src/GamePackage/Die.java

```diff
@@ -0,0 +1,53 @@
+package GamePackage;
+
+import java.util.Random;
+
+/*****************************************************************
+ The Die Class will be a random number generator that "rolls"
+ a die some methods to return the value of the die.
+
+ @author Dylan Kernohan
+ @version Winter 2018
+ *****************************************************************/
+public class Die {
+
+    /** current value of the die */
+    int value;
+
+    /*****************************************************************
+     Default constructor
+     *****************************************************************/
+    public Die( ) {
+    }
+
+    /*****************************************************************
+     Set the die value
+     @param value
+     @return none
+     *****************************************************************/
+    public void setValue(int value){
+        this.value = value;

```

Comment:
Do we think it is necessary to make sure that set value doesn't accept integers outside of the range of 1-6?

Reply:
I do not think so, the setValue function is simply for us to use for testing and stuff. The purpose of it is to assign a value. We may want to assign values bigger than 6 to test our validation cases or something. Make sense?